apps/dashboard/src/components/primitives/field-editor/variable-plugin/variable-theme.ts

```diff
@@ -0,0 +1,61 @@
+import { EditorView } from '@uiw/react-codemirror';
+import { VARIABLE_PILL_CLASS } from './constants';
+
+export const variablePillTheme = EditorView.baseTheme({
+  [`.${VARIABLE_PILL_CLASS}`]: {
+    backgroundColor: 'hsl(var(--bg-weak))',
+    color: 'hsl(var(--text-sub))',
+    border: '1px solid hsl(var(--stroke-soft))',
+    borderRadius: '10px',
+    padding: '2px 6px 2px 6px',
+    fontFamily: 'inherit',
+    display: 'inline-flex',
+    alignItems: 'center',
+    height: '16px',
+    lineHeight: '16px',
+    fontSize: '12px',
```

Comment:
Do we have font and typography-related CSS root variables to use as we did with colors?

Reply:
nope, currently all of the definitions are managed inside of the tailwind config file only. I will take a look at this at later stage as part of the new tailwind version upgrade, where they are moving away from the tailwind config file into a CSS variable native approach.